watch-history-exporter-for-amazon-prime-video.js

```diff
@@ -1,3 +1,22 @@
+function germanDateToISO(germanDate) {
+    const months = {
+        "Januar": 0, "Februar": 1, "März": 2, "April": 3, "Mai": 4, "Juni": 5,
+        "Juli": 6, "August": 7, "September": 8, "Oktober": 9, "November": 10, "Dezember": 11
+    };
+
+    const dateParts = germanDate.match(/^(\d{1,2})\. ([A-Za-zäöüÄÖÜß]+) (\d{4})$/);
+    if (!dateParts) throw new Error("Invalid German date format");
+
+    const day = parseInt(dateParts[1], 10);
+    const month = months[dateParts[2]];
+    const year = parseInt(dateParts[3], 10);
+
+    if (month === undefined) throw new Error("Invalid month name");
+
+    const date = new Date(Date.UTC(year, month, day));
```

Comment:
Do we need to use `Date.UTC` here? I'm not sure if the dates on the page are in UTC or a local timezone, but I would just use `Date(year, month, day)` unless we need to do this.